nuxeo-coldstorage/src/main/java/org/nuxeo/coldstorage/helpers/ColdStorageHelper.java

```diff
@@ -252,7 +267,7 @@ public int getTotalAvailable() {
         }
     }
 
-    private ColdStorageHelper() {
+    public ColdStorageHelper() {
         // no instance allowed
```

Comment:
The point of these kinds of helpers is to not be instantiated, so this should stay `private`, as the comment says.